README.md

```diff
@@ -52,8 +52,9 @@ community.
 3. Review should result in the DIP either being moved to `Approved` status or
    modified with a list of issues that need to be worked on before a final
    decision can be made. In case the DIP topic seems important but language
-   authors decide it needs more research, a new topic in dlang-study@puremagic.com
-   mail list may be initiated.
+   authors decide it needs more research, a new topic on the [NG](http://forum.dlang.org/group/general)
```

Comment:
unfortunately dlang-study is a bit inactive atm, hence the proposal to add a link to the NG